src/helpers.ts

```diff
@@ -0,0 +1,39 @@
+export const sortByAsc = (data: any, type: string) => {
```

Comment:
```suggestion
export const sortByAsc = (data: any, type: string) => {
```

This `any` usage concerns me as this defeats the purpose of `typescript`. We need to find a better and strong typed way to do it